### PR TITLE
add more scala versions in scripted test

### LINKED ISSUE #629: Which Scala versions are supported
There are scala `2.10` and `2.11` in the `app` module `crossScalaVersions`.

https://github.com/eed3si9n/scalaxb/blob/85271ccb1c6e5f7d779152a13994d7e2e520da3e/build.sbt#L57

But maybe does not check in github actions CI.

https://github.com/eed3si9n/scalaxb/blob/85271ccb1c6e5f7d779152a13994d7e2e520da3e/build.sbt#L77
https://github.com/eed3si9n/scalaxb/blob/85271ccb1c6e5f7d779152a13994d7e2e520da3e/.github/workflows/ci.yml#L35-L37

- Should we fix ci settings?(restore scala 2.10 and 2.11 test)
- remove old scala and sbt 0.13 version settings in build files

WDYT? 👀 🤔

### PR DESCRIPTION
**Pull request by xuwei-k**
- fix https://github.com/eed3si9n/scalaxb/issues/629

### CODE REVIEW

File: sbt-scalaxb/src/sbt-test/sbt-scalaxb/address/build.sbt [hunk 1]
```diff
@@ -1,13 +1,38 @@
...
 lazy val root = (project in file(".")).
   enablePlugins(ScalaxbPlugin).
   settings(
-    scalaVersion := "2.12.18",
+    crossScalaVersions := Seq(
+      "2.10.7",
+      "2.11.12",
+      "2.12.18",
+      "2.13.12",
+    ),
```
Comment (xuwei-k) [inline sbt-scalaxb/src/sbt-test/sbt-scalaxb/address/build.sbt:32]: could not add Scala 3 due to https://github.com/eed3si9n/scalaxb/blob/64033cfa3ce33591815eb9907f84b648b95301bd/build.sbt#L98

(sbt 1.2 does not support Scala 3)

Comment (eed3si9n) [reply sbt-scalaxb/src/sbt-test/sbt-scalaxb/address/build.sbt:32]: Let's use sbt 1.5.8 then

Comment (xuwei-k) [reply sbt-scalaxb/src/sbt-test/sbt-scalaxb/address/build.sbt:32]: done

**Comment by eed3si9n** [comment]
Thanks!

**Pull request merged**